extensions/skyportal/static/js/components/Group.jsx

```diff
@@ -0,0 +1,598 @@
+  };
+
+  const handlePanelMembersChange = (panel) => (event, isExpanded) => {
+    setPanelMembersExpanded(isExpanded ? panel : false);
+  };
+  const handlePanelStreamsChange = (panel) => (event, isExpanded) => {
+    setPanelStreamsExpanded(isExpanded ? panel : false);
+  };
+
+  const { id } = useParams();
+  const loadedId = useSelector((state) => state.group.id);
+
+  useEffect(() => {
+    const fetchGroup = async () => {
+      const data = await dispatch(groupActions.fetchGroup(id));
+      if (data.status === "error") {
+        setGroupLoadError(data.message);
+      }
+    };
+    fetchGroup();
+  }, [id, loadedId, dispatch]);
+
+  const group = useSelector((state) => state.group);
+  const currentUser = useSelector((state) => state.profile);
+
+  // fetch streams:
+  const streams = useSelector((state) => state.streams);
+
+  useEffect(() => {
+    const fetchStreams = async () => {
+      const data = await dispatch(streamsActions.fetchStreams());
+      if (data.status === "error") {
+        setGroupLoadError(data.message);
+      }
+    };
+    fetchStreams();
+  }, [currentUser, dispatch]);
+
+  // forms
+  // add stream to group
+  const onSubmitAddStream = async (data) => {
+    const result = await dispatch(
+      streamsActions.addGroupStream({
+        group_id: group.id,
+        stream_id: data.stream_id,
+      })
+    );
+    if (result.status === "success") {
+      dispatch(showNotification("Added stream to group"));
+    }
+    setAddStreamOpen(false);
+  };
+  // add filter to group
+  const onSubmitAddFilter = async (data) => {
+    const result = await dispatch(
+      filterActions.addGroupFilter({
+        name: data.filter_name,
+        group_id: group.id,
+        stream_id: data.filter_stream_id,
+      })
+    );
+    if (result.status === "success") {
+      dispatch(showNotification("Added filter to group"));
+    }
+    handleDialogClose();
+    dispatch(groupActions.fetchGroup(loadedId));
+  };
+
+  if (groupLoadError) {
+    return <div>{groupLoadError}</div>;
+  }
+
+  // renders
+  if (!group) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+
+  // currentUser may not have the "Group admin" role, but can still be the group admin?
+  const currentGroupUser = group?.users?.filter(
+    (group_user) => group_user.username === currentUser.username
+  )[0];
+
+  const isAdmin = (aUser, aGroup) =>
+    aUser &&
+    aGroup.group_users &&
+    aGroup.group_users.filter(
+      (group_user) => group_user.user_id === aUser.id
+    )[0].admin;
+
+  let numAdmins = 0;
+  group?.group_users?.forEach((groupUser) => {
+    if (groupUser?.admin) {
+      numAdmins += 1;
+    }
+  });
+
+  const groupStreamIds = group?.streams?.map((stream) => stream.id);
+
+  const isStreamIdInStreams = (sid) =>
+    streams?.map((stream) => stream.id).includes(sid);
+
+  return (
+    <div>
+      <Typography variant="h5" style={{ paddingBottom: 10 }}>
```

Comment:
```suggestion
      <Typography variant="h6" style={{ paddingBottom: 10 }}>
```
My PR for the front page changes the front page widget titles to variant h6 so we can go for that smaller overall font feel we discussed. You may want to do the same here.

Reply:
Fixed